pymeasure/instruments/attocube/anc300.py

```diff
@@ -255,3 +246,109 @@ def stop_all(self):
             attribute = getattr(self, attr)
             if isinstance(attribute, Axis):
                 attribute.stop()
+
+    def __init__(
+        self,
+        adapter=None,
+        name="attocube ANC300 Piezo Controller",
+        axisnames="",
+        passwd="",
+        query_delay=0.05,
+        **kwargs,
+    ):
+        """
+        The function is called with the following arguments:
+
+        The function checks if the adapter is None, and if so, it checks if the host is defined.
+        If the host is defined, it creates a VISA socket connection.
+
+        .. deprecated:: 0.11.2
+            The 'host' argument is deprecated. Use 'adapter' instead.
+
+        :param adapter: The address of the controller
+        :param name: The name of the device, defaults to attocube ANC300 Piezo Controller (optional)
+        :param axisnames: a string of the axis names, e.g. "xyz"
+        :param passwd: the password for the controller
+        :param query_delay: the time to wait between queries to the controller
+        """
```

Comment:
The class docstring is shown for the init as well, please move all relevant information from here to the class docstring and do not add any docstring to the init (such that the init gets the class docstring per default).

Reply:
Understood, and done. Thanks.